src/shared/bootspec.c

```diff
@@ -591,8 +591,8 @@ static int boot_entries_select_default(const BootConfig *config) {
         assert(config->entries || config->n_entries == 0);
 
         if (config->n_entries == 0) {
-                log_debug("Found no default boot entry :(");
-                return -1; /* -1 means "no default" */
+                return log_debug_errno(SYNTHETIC_ERRNO(1),
+                                       "Found no default boot entry :("); /* -1 means "no default" */
         }
```

Comment:
Parens can be dropped.

Reply:
This is very ugly. -1 is not an errno code but an index here. I'd prefer to leave this as it was. Sorry for not noticing this earlier.